src/BIMDataComponents/BIMDataPaginatedList/BIMDataPaginatedList.vue

```diff
@@ -104,11 +110,51 @@ export default {
       deep: true,
       immediate: true,
     },
+    activeElement: {
+      handler(value) {
+        if (value) {
+          this.showElement(value);
+        }
+      },
+      immediate: true,
+    },
   },
   methods: {
     onPageChange(pageNumber) {
       this.currentPage = pageNumber;
     },
+    compareElements(elementA, elementB) {
+      if (this.elementKey) {
+        return elementA[this.elementKey] === elementB[this.elementKey];
+      } else {
+        return elementA === elementB;
+      }
+    },
+    isElementActive(element) {
+      if (!this.activeElement) return false;
+      return this.compareElements(element, this.activeElement);
+    },
+    async showElement(elem) {
+      const element = this.list.find(listElement =>
+        this.compareElements(elem, listElement)
+      );
+      if (!element) return;
+      const elementIndex = this.list.findIndex(listElement =>
+        this.compareElements(elem, listElement)
+      );
```

Comment:
You are doing the same thing twice here. This could be replaced by:
```js
const elementIndex = this.list.findIndex(listElement =>
  this.compareElements(elem, listElement)
);
if (elementINdex === -1) return;
...
```